src/Aks/Aks/Commands/NewAzureRmAks.cs

```diff
@@ -102,6 +103,7 @@ public class NewAzureRmAks : CreateOrUpdateKubeBase
         [Parameter(Mandatory = false, HelpMessage = "DNS service IP used for building Kubernetes network.")]
         public string DnsServiceIP { get; set; }
 
+        [CmdletParameterBreakingChange("DockerBridgeCidr", ChangeDescription = "DockerBridgeCidr parameter is being deprecated without being replaced.")]
```

Comment:
"will be deprecated" on ${future Az version}

Reply:
Updated. 
The warning message will be:
```powershell
WARNING: Upcoming breaking changes in the cmdlet 'New-AzAksCluster' :
- The parameter : 'DockerBridgeCidr' is changing.
- Change description : DockerBridgeCidr parameter will be deprecated in Az 11.0.0 without being replaced.
- The change is expected to take effect from the version : '6.0.0'
Note : Go to https://aka.ms/azps-changewarnings for steps to suppress this breaking change warning, and other information on breaking changes in Azure PowerShell.
```